resources/repos.json

```diff
@@ -1839,4 +1839,22 @@
             }
         }
     }
+    "LordIdra/Quaptics:master": {
```

Comment:
Nevermind, the json validator caught a missing comma here

Reply:
Oop... my bad, fixed now.